language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaWorkspaceService.java

```diff
@@ -187,7 +187,8 @@ public void setExperimentalClientCapabilities(Map<String, Boolean> experimentalC
      * Experimental capabilities.
      */
     public enum Experimental {
-        INTROSPECTION("introspection"), SHOW_TEXT_DOCUMENT("showTextDocument");
+        INTROSPECTION("introspection"), SHOW_TEXT_DOCUMENT("showTextDocument"),
+        SEMANTIC_SYNTAX_HIGHLIGHTER("semanticSyntaxHighlighter");
```

Comment:
Let's get all these experimental operations to a common Enum instead of a nested Enum